### PR TITLE
#169817530 Integrate HoundCI 

### PR DESCRIPTION
**Pull request by erickyvand**
#### What does this PR do?
- Integrate HoundCi and put the badge on readme file
#### What are the relevant pivotal tracker stories?
- [#169817530](https://www.pivotaltracker.com/story/show/169817530)
#### Screenshots
Screen Shot 2019-11-28 at 20 59 09

### CODE REVIEW

**Comment by adesege** [comment]
You still have unresolved merged conflicts

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (adesege) [inline .eslintrc.json]: Remove this file and other files that are not related to this task

**Comment by adesege** [comment]
You have merge conflicts. Please rebase with develop

File: .eslintrc.json (hunk 1 above)
Comment (adesege) [reply .eslintrc.json]: @erickyvand ☝️

File: package.json [hunk 2]
```diff
@@ -8,14 +8,18 @@
...
+  "author": "snipers",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/andela/blackninjas-backend/issues"
+    "url": "https://github.com/andela/the_spinners-backend/issues"
+  },
+  "homepage": "https://github.com/andela/the_spinners-backend#readme",
+  "devDependencies": {
+    "eslint": "^6.7.1",
+    "coveralls": "^3.0.9"
```
Comment (adesege) [inline package.json]: Remove this package.

File: .gitignore [hunk 3]
```diff
@@ -2,4 +2,5 @@
 node_modules
 coverage
 package-lock.json
-.env
\ No newline at end of file
+.env
```
Comment (adesege) [inline .gitignore]: Remove this line. It has nothing to do with HoundCI integration

File: .hound.yml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+eslint:
```
Comment (adesege) [inline .hound.yml:1]: How do I confirm that this works?

**Review by adesege — CHANGES_REQUESTED** [review]

File: .eslintrc.json (hunk 1 above)
Comment (erickyvand) [reply .eslintrc.json]: These files were added when I did rebase from Develop, that is why I was asking do I need to delete it again?

And as you told me I'm waiting to rebase again

File: .hound.yml (hunk 4 above)
Comment (adesege) [reply .hound.yml:1]: @erickyvand ☝️

Comment (erickyvand) [reply .hound.yml:1]: I saw in hound documentation that you have to add enable: true, and add .eslintrc so that hound can work with eslint.

Comment (adesege) [reply .hound.yml:1]: Ok. Can you resolve the conflict you have?

Comment (erickyvand) [reply .hound.yml:1]: Okay let me solve it
